using_images/s2i_images/nodejs.adoc

```diff
@@ -136,3 +149,20 @@ $ oc rsh <pod_id>
 
 Entering into a running container changes your current directory to
 *_/opt/app-root/src_*, where the source code is located.
+
+ifdef::openshift-online[]
+[[nodejs-templates]]
+== Node.js Templates
+
+{product-title} includes an example template to deploy a
+link:https://github.com/openshift/nodejs-ex[sample Node.js application].
+This template builds and deploys the sample application on Node.js with a
+MongoDB database using a persistent volume for storage.
+
+The sample application can be built and deployed using the
+`rhscl/nodejs-4-rhel7` image with the following command:
+
+----
+oc new-app --template=nodejs-mongo-persistent
```

Comment:
@Miciah Thank you! We need the command prompt $ here, but that's my only comment. Thanks again!